softhddevice.cpp

```diff
@@ -1242,7 +1242,7 @@ void cMenuSetupSoft::Create(void)
 	Add(new cMenuEditBoolItem(tr("60hz display mode"), &_60HzMode,
 		trVDR("no"), trVDR("yes")));
 	Add(new cMenuEditStraItem(tr("A/V sync start"),
-		 &SoftStartSync, 4, video_sync_mode));
+		 &SoftStartSync, sizeof(video_sync_mode), video_sync_mode));
```

Comment:
&SoftStartSync, sizeof(video_sync_mode), video_sync_mode));
I have a segfault and null in menu when selecting mode.
with 
&SoftStartSync, 5, video_sync_mode));
all ok.